### PR TITLE
modified button component in patient register

### PR DESCRIPTION
**Pull request by akif00**
buttonr-1
buttonr-2
buttonr-3
buttonr-4
M
buttonr-5
buttonr-6

### CODE REVIEW

File: src/patient/pages/Appointments/appointments.module.css [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+/* .ptName input {
+    margin-top: 5px;
+    padding: 10px 10px;
+    border-radius: 5px;
+    border: 1px solid #7c7b7b7e;
+    transition-duration: 300ms;
+    width: 100%;
+    max-width: 500px;
+} */
```
Comment (tajammul1996) [inline src/patient/pages/Appointments/appointments.module.css:35]: can you remove this comment

File: src/patient/pages/Appointments/appointments.module.css [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+
+/* .ptAge input:focus {
+    border: 1px solid rgb(19, 63, 92);
+} */
```
Comment (tajammul1996) [inline src/patient/pages/Appointments/appointments.module.css:64]: this too

File: src/patient/pages/Appointments/appointments.module.css [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+    border-radius: 5px;
+    border: 1px solid #7c7b7b7e;
+    transition-duration: 300ms;
+    width: 100%;
+    max-width: 420px;
+}
+
+.ptphone input:focus {
+    border: 1px solid rgb(19, 63, 92);
+} */
```
Comment (tajammul1996) [inline src/patient/pages/Appointments/appointments.module.css:92]: this too

File: src/patient/pages/Appointments/appointments.module.css [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    font-size: 1.1rem;
+    margin-right: 20px;
+}
+
+
+/* .ptYesNo input.yesCheck {
+    width: 18px;
+    height: 18px;
+    margin-right: 5px;
+} */
```
Comment (tajammul1996) [inline src/patient/pages/Appointments/appointments.module.css:152]: this too and other all

File: src/shared/components/Input/input.module.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    transition-duration: 300ms;
+    width: 90%;
+    max-width: 500px;
+    /* padding: 5px;
+    border-radius: 5px;
+    border: 1px solid #7c7b7b7e;
+    transition-duration: 300ms;
+    width: 100%;
+    max-width: 500px;
+    margin: 8px; */
```
Comment (tajammul1996) [inline src/shared/components/Input/input.module.css:16]: this too

File: src/patient/pages/Appointments/appointments.module.css (hunk 1 above)
Comment (akif00) [reply src/patient/pages/Appointments/appointments.module.css:35]: on it

**Pull request merged**